restore.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source /pgenv.sh
+
+echo "TARGET_DB: ${TARGET_DB}"
+echo "WITH_POSTGIS: ${WITH_POSTGIS}"
+echo "TARGET_ARCHIVE: ${TARGET_ARCHIVE}"
+
+if [ -z "${TARGET_ARCHIVE:-}" ] || [ ! -f "${TARGET_ARCHIVE:-}" ]; then
+	echo "TARGET_ARCHIVE needed."
+	exit 1
+fi
+
+if [ -z "${TARGET_DB:-}" ]; then
+	echo "TARGET_DB needed."
+	exit 1
+fi
+
+echo "Dropping target DB"
+dropdb ${TARGET_DB}
+
+if [ -z "${WITH_POSTGIS:-}" ]; then
+	echo "Recreate target DB without POSTGIS"
+	createdb -O ${PGUSER} ${TARGET_DB}
+else
+	echo "Recreate target DB with POSTGIS"
+	createdb -O ${PGUSER} -T template_postgis ${TARGET_DB}
```

Comment:
@lucernae we are trying to do away with the template method for installing PostGIS see https://github.com/kartoza/docker-postgis/pull/119